src/olympia/git/tests/test_utils.py

```diff
@@ -1502,3 +1506,62 @@ def test_is_recent_with_oldish_repo(settings):
     update_git_repo_creation_time(repo, time=time)
 
     assert not repo.is_recent
+
+
+@pytest.mark.django_db
+@mock.patch('olympia.git.utils.statsd.timer')
+@mock.patch('olympia.git.utils.statsd.incr')
+def test_extract_version_to_git(incr_mock, timer_mock):
```

Comment:
Just wondering, how does it know what `incr_mock` and `timer_mock` are? Is it based on the position of the `@mock.patch` calls? If so, are they reversed here, or is the first argument the last mock defined?

Reply:
It is indeed based on the position of the `@mock.patch` calls. They are inverted because of the way decorators are called. We could expand this snippet like this:

```
patch_timer(patch_incr(test_extract_version_to_git))
```

so `test_extract_version_to_git` receives the `incr` mock first, and then the `timer` one.